pages/components/Rating/rating.html

```diff
@@ -0,0 +1,137 @@
+            </li>
+            <li class="component-links">
+                <a href="../List/list.html">List</a>
+            </li>
+            <li class="component-links">
+                <a href="../Navigation/navigation.html">Navigation</a>
+            </li>
+            <li class="component-links">
+                <a href="../Toast/toast.html">Toast</a>
+            </li>
+            <li class="component-links">
+                <a href="../Text/text.html">Text</a>
+            </li>
+            <li class="component-links">
+                <a href="../Modal/modal.html">Modal</a>
+            </li>
+            <li class="component-links">
+                <a href="../Grid/grid.html">Grid</a>
+            </li>
+            <li class="component-links">
+                <a href="../Rating/rating.html">Rating</a>
+            </li>
+        </ul>
+    </nav>
+    <main class="component-display">
+        <h1 class="component-display-header">Ratings</h1>
+        <pre class="comp-code-container"><code class="html">
+    @import url("https://si-mpleui.netlify.app/pages/components/Rating/rating.css");
+    &lt;link rel="stylesheet" 
+    href="https://si-mpleui.netlify.app/pages/components/Rating/rating.css">
+            </code></pre>
+        <h2 class="sub-comp-header">Poor Rating</h2>
+        <span class="fa fa-star checked-poor"></span>
+        <span class="fa fa-star checked-poor"></span>
+        <span class="fa fa-star empty"></span>
+        <span class="fa fa-star empty"></span>
+        <span class="fa fa-star empty"></span>
+        <br />
+        <h2 class="sub-comp-header">Moderate Rating</h2>
+        <span class="fa fa-star checked-moderate"></span>
+        <span class="fa fa-star checked-moderate"></span>
+        <span class="fa fa-star checked-moderate"></span>
+        <span class="fa fa-star empty"></span>
+        <span class="fa fa-star empty"></span>
+        <br />
+        <h2 class="sub-comp-header">Good Rating</h2>
+        <span class="fa fa-star checked-good"></span>
+        <span class="fa fa-star checked-good"></span>
+        <span class="fa fa-star checked-good"></span>
+        <span class="fa fa-star checked-good"></span>
+        <span class="fa fa-star checked-good"></span>
+        <br />
+        <p class="component-description">
+            A Rating component can be used with the following syntax.
+        </p>
+        <pre class="comp-code-container">
+            <code class="html">
+                &lt;span class="fa fa-star checked-poor">&lt;/span>
+                &lt;span class="fa fa-star checked-moderate">&lt;/span>
+                &lt;span class="fa fa-star checked-good">&lt;/span>
+            </code>
+        </pre>
+        <footer class="page-footer">
+            <a href="../Grid/grid.html" class="footer-button">
+                <img src="../../../images/previous-arrow.svg" alt="" />Grid
+            </a>
+            <!-- <a href="../Text/text.html" class="footer-button">
```

Comment:
You can remove these. If you are not going to use them in future.

Reply:
This is a tailing Page, I will be using this for any new component I add to the library. Hence the code is commented.